scripts/Start.ps1

```diff
@@ -8,6 +8,20 @@ $FrontendScript = Join-Path "$PSScriptRoot" 'Start-Frontend.ps1'
 
 # Start backend (in new PS process)
 Start-Process pwsh -ArgumentList "-command $BackendScript"
+# check if the backend is running before proceeding
+$backendRunning = $false
+
+# get the port from the REACT_APP_BACKEND_URI env variable
+$envFilePath = Join-Path $PSScriptRoot '..\webapp\.env'
+$envContent = Get-Content -Path $envFilePath
+# Write-Output $envContent
```

Comment:
Comments at lines 17 and 19 should be removed. Please do so in another commit, not another branch! ;)